crates/air_r_formatter/src/r/auxiliary/binary_expression.rs

```diff
@@ -211,11 +218,23 @@ fn fmt_binary_assignment(
             left.format(),
             space(),
             operator.format(),
-            right
+            right_formatted
         ])]
     )
 }
 
+fn format_assignment_rhs(expr: &AnyRExpression, table: bool) -> impl Format<RFormatContext> {
+    format_with(move |f| {
+        if table {
+            if let AnyRExpression::RCall(call) = expr {
+                let options = FormatRCallArgumentsOptions { table: true };
+                return write!(f, [call.format().with_options(options)]);
+            }
+        }
+        write!(f, [expr.format()])
+    })
+}
```

Comment:
```suggestion
fn format_assignment_rhs(right: &AnyRExpression, table: bool) -> impl Format<RFormatContext> {
    format_with(move |f| {
        if table {
            if let AnyRExpression::RCall(call) = right {
                let options = FormatRCallArgumentsOptions { table: true };
                return write!(f, [call.format().with_options(options)]);
            }
        }
        write!(f, [right.format()])
    })
}
```

Because `binary_assignment_has_persistent_line_break()` also uses `right`, so it's slightly more consistent I think, and a little easier to read on its own